api/envoy/extensions/filters/http/ext_proc/v3/ext_proc.proto

```diff
@@ -328,6 +329,11 @@ message ExternalProcessor {
   // the ``allowed_override_modes`` allow-list below.
   // Since request_header_mode is not applicable in any way, it's ignored in comparison.
   repeated ProcessingMode allowed_override_modes = 22;
+
+  // Decorator to introduce custom logic that runs after a message received from
+  // the External Processor is processed.
+  // [#extension-category: envoy.http.ext_proc.on_processing_response]
```

Comment:
Would `envoy.http.ext_proc.response_processors` be a better category name?

Reply:
Just looked below, and it seems you do use the correct category name, so let's just update this line.